browser/brave_vpn/win/brave_vpn_wireguard_service/allowlist.gni

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Allowlist file used by brave resources, created after merging chrome allowlist
+# and brave resources.
+brave_allowlist_file = "${root_gen_dir}/brave_resources_allowlist.txt"
+
+# Brave VPN Wireguard service allowlist file contains resource ids from
+# brave_vpn_wireguard_service.exe.
+wireguard_resource_allowlist = "$root_gen_dir/wireguard_resources_allowlist.txt"
```

Comment:
`wireguard_resource_allowlist_file` to differentiate it from the `wireguard_resource_allowlist` target

Reply:
renamed